book/background_task.md

```diff
@@ -1,34 +1,77 @@
-# Background Tasks with Nu
+# Background Jobs
 
-Currently, Nushell doesn't have built-in background task management feature, but you can make it "support" background task with some tools, here are some examples:
+## Spawning Jobs
 
-1. Using a third-party task management tools, like [pueue](https://github.com/Nukesor/pueue)
-2. Using a terminal multiplexer, like [tmux](https://github.com/tmux/tmux/wiki) or [zellij](https://zellij.dev/)
+Nushell currently presents experimental support for thread-based background jobs.
```

Comment:
could you move this line to above the "Spawning Jobs" header?

also, nitpick, but not sure about "presents" here, I think just "has" works fine